README.rst

```diff
@@ -1,9 +1,24 @@
 Kagi
 ----
 
-.. image:: https://circleci.com/gh/justinmayer/kagi.svg?style=svg
+|coc| |readthedocs| |pypi| |circleci|
+
+
+.. |coc| image:: https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-blue.svg
+    :target: https://github.com/justinmayer/kagi/blob/master/CODE_OF_CONDUCT.rst
+    :alt: Code of conduct
+
+.. |circleci| image:: https://circleci.com/gh/justinmayer/kagi.svg?style=svg
     :target: https://circleci.com/gh/justinmayer/kagi
```

Comment:
I've never really liked the aesthetics of the badge provided by CircleCI, and the included brand logo is a bit superfluous. What if we used a shield-style badge instead? Then they would all match. Perhaps something like:
```
.. |circleci| image:: https://img.shields.io/circleci/build/github/justinmayer/kagi
    :target: https://circleci.com/gh/justinmayer/kagi
    :alt: Build Status
```